parser/charset/encoding_gbk.go

```diff
@@ -91,3 +92,41 @@ var GBKCase = unicode.SpecialCase{
 	unicode.CaseRange{Lo: 0x01DC, Hi: 0x01DC, Delta: [unicode.MaxCase]rune{0, 0, 0}},
 	unicode.CaseRange{Lo: 0x216A, Hi: 0x216B, Delta: [unicode.MaxCase]rune{0, 0, 0}},
 }
+
+// customGBK is a simplifiedchinese.GBK wrapper
+type customGBK struct{}
+
+// NewDecoder returns simplifiedchinese.GBK.NewDecoder()
```

Comment:
```suggestion
// NewDecoder returns simplifiedchinese.GBK.NewDecoder().
```
Please add `.` at the end of a sentence.